packages/api-v2/src/AuthProvider.tsx

```diff
@@ -81,14 +81,20 @@ const AuthProvider = ({ children, cookieTimeout }: AuthProviderProps) => {
 
     const { mutateAsync } = useMutation('authorize');
 
-    const { queryClient } = useAPIContext();
+    const { queryClient, setOnReconnected } = useAPIContext();
 
     const [isLoading, setIsLoading] = useState(true);
     const [isSuccess, setIsSuccess] = useState(false);
     const [isError, setIsError] = useState(false);
     const [isFetching, setIsFetching] = useState(false);
     const [data, setData] = useState<TSocketResponseData<'authorize'> | null>();
 
+    useEffect(() => {
+        setOnReconnected(() => {
+            mutateAsync({ payload: { authorize: getToken(loginid || '') || '' } });
```

Comment:
Seen this being used instead of the `||` not sure if it applies here aswell
```suggestion
            mutateAsync({ payload: { authorize: getToken(loginid || '') ?? '' } });
```

Reply:
updated :)
I really like the comments with "commit suggestion" which enables me to just commit stuff.